packages/docs/src/page-configs/ui-elements/icon/examples/Default.vue

```diff
@@ -2,39 +2,25 @@
   <div>
       <p>
         Material Design Icons<br>
-        <va-icon class="material-icons">home</va-icon>
-        <va-icon name="home" />
-      </p>
-
-      <p>
-        Font Awesome 4<br>
-        <va-icon class="fa fa-home" />
-        <va-icon name="fa4-home" />
+        <va-icon class="material-icons">book</va-icon>
+        <va-icon name="book" />
       </p>
 
       <p>
         Font Awesome 5<br>
-        <va-icon class="fas fa-home" />
-        <va-icon name="fas-home" />
-      </p>
-
-      <p>
-        Entypo<br>
-        <va-icon class="entypo-home" />
-        <va-icon name="entypo-home" />
+        <va-icon class="fas fa-book" />
+        <va-icon name="fas-book" />
       </p>
 
       <p>
         Ionic<br>
-        <va-icon class="icon ion-md-home" />
-        <va-icon name="ion-home" />
+        <va-icon name="ion-book-outline" />
+        <va-icon name="ion-book" />
```

Comment:
The idea with 2 icons was:
* for first icon - we use attributes (class or ligature + class)
* for second item we use `name`

Here both use `name`.

First one should be something like `<va-icon name="ion-book" tag="ion-icon">`

Reply:
@asvae updated example.

Reply:
<va-icon name="ion-book-outline" /> ?